tasks/image-segmentation-yolov8/task.yaml

```diff
@@ -0,0 +1,68 @@
+id: image-segmentation-yolov8
+name: image-segmentation-yolov8
+description:
+is_active: true
+project_type: 1
+param_groups:
+- name: inputs
+  params:
+  - name: dataset
+    description: Dataset that is used for fetching dataset from coretex.
+    value: 2022
+    data_type: dataset
+    required: true
+  - name: trainedModel
+    description: Model used for image segmentation that needs validation
+    value: 701
+    data_type: model
+    required: false
+- name: outputs
+  params:
+  - name: model
+    description: Model trained by this Task
+    value: null
+    data_type: model
+    required: false
+- name: sharedParameters
+  params:
+  - name: validationMode
+    description: Whether to perform training or validation for image segmentation. If true, only validation
+      will be performed; if false, only training will be performed
+    value: false
+    data_type: bool
+    required: true
+  - name: batchSize
+    description: Number of segments into which dataset is split during the training
+    value: 16
+    data_type: int
+    required: true
+  - name: excludedClasses
```

Comment:
`excludedClasses` should not be a shared parameter, it is a training-only parameter.